src/types/subgraph/codegen.ts

```diff
@@ -0,0 +1,23 @@
+import { CodegenConfig } from '@graphql-codegen/cli';
+
+const config: CodegenConfig = {
+  schema: process.env.SUBGRAPH_GRAPHQL_ENDPOINT,
+  documents: ['src/clients/subgraph/**/*.graphql'],
+  generates: {
+    './src/types/subgraph/gql/queries.ts': {
+      plugins: [
+        {
+          add: {
+            placement: 'append',
+            content: `export const subgraphEndpoint = '${process.env.SUBGRAPH_GRAPHQL_ENDPOINT}';`,
+          },
+        },
+        'typescript',
+        'typed-document-node',
+        'typescript-operations',
+      ],
+    },
+  },
+};
+
+export default config;
```

Comment:
I'd move this file and the generated types inside the subgraph client folder (`src/clients/subgraph`) then create an index file that would export all the generated types and documents as well as the subgraph client. This way everything would be contained within the same module and can be imported using one absolute import.

```ts
import { subgraphClient, AssetsInAccountQueryDocument } from 'clients/subgraph';
```